packages/components/src/ui/data-table-filter/lib/memo.ts

```
@@ -0,0 +1,35 @@
+export function memo<TDeps extends readonly any[], TResult>(
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)